### DISCUSSION TITLE
How do I insert a node into existing markdown and save the new markdown?

### DISCUSSION THREAD

**Discussion by darrenparkinson** [yuin/goldmark#362] [Q&A] [answered]
I have some markdown content that I'd like to insert a table into after the first header. 

I don't want to render HTML, I just would like to get the resulting markdown as a []byte that I can then save back into a file.

Any assistance greatly appreciated.

Many thanks.

**Comment by abhinav** [comment, marked as answer]
You'll need a renderer that supports generating Markdown.
goldmark does not provide one natively at this time.

I'm aware of the following third-party implementations; there may be others.

- https://github.com/Kunde21/markdownfmt/
- https://github.com/teekennedy/goldmark-markdown
